src/sqlfluff/dialects/dialect_spark3.py

```diff
@@ -1645,22 +1645,20 @@ class PivotClauseSegment(BaseSegment):
             ),
             "IN",
             Bracketed(
-                OneOf(
+                Delimited(
                     Sequence(
                         OneOf(
-                            Ref("ScalarValue"),
-                            Ref("TupleValue"),
-                        ),
-                        Ref("AliasExpressionSegment", optional=True),
-                    ),
-                    Delimited(
-                        Sequence(
-                            OneOf(
-                                Ref("ScalarValue"),
-                                Ref("TupleValue"),
+                            Bracketed(
+                                Delimited(
+                                    Ref("ExpressionSegment"),
+                                    ephemeral_name="ValuesClauseElements",
+                                )
+                            ),
+                            Delimited(
+                                Ref("ExpressionSegment"),
```

Comment:
Whoever did this clause used the segments for the Spark3 values clause, which should be made consistent with all other dialects.
cleaning up this reference